src/prefect/orion/schemas/states.py

```diff
@@ -69,6 +69,13 @@ class Config:
             "Content must be storable as JSON."
         ),
     )
+    _data: Optional[Any] = Field(
```

Comment:
Hmm do we need this in the api schema? I thought `_data` shouldn't be present in responses